doc/api/fs.markdown

```diff
@@ -528,13 +530,21 @@ If `options` is a string, then it specifies the encoding. Example:
 
     fs.writeFile('message.txt', 'Hello Node.js', 'utf8', callback);
 
-## fs.writeFileSync(filename, data[, options])
+Any specified file descriptor has to support writing.
```

Comment:
Whether data will be appended depends on the flags that the `fd` was created with. The `a` flags will obviously append, but the `w` flags will create/truncate instead.